q2_feature_classifier/_consensus_assignment.py

```diff
@@ -99,13 +101,11 @@ def _blast6format_df_to_series_of_lists(
             <accession ID>  Annotation
         The accession IDs in this taxonomy should match the subject-seq-ids in
         the "assignment" input.
-    '''
-    taxa_hits = assignments.set_index('qseqid')['sseqid']
-
+    """
     # validate that assignments are present in reference taxonomy
     # (i.e., that the correct reference taxonomy was used).
     # Note that we drop unassigned labels from this set.
-    missing_ids = set(taxa_hits.values) - set(ref_taxa.index) - {'*', ''}
+    missing_ids = set(assignments['sseqid'].values) - set(ref_taxa.index) - {'*', ''}
```

Comment:
My bad, I'm used to Black that tends to be a bit more liberal than the pep 8

`pandas_obj.unique()` might be used and should not change the results but won't it need to still be wrapped in `set()` in order to work ? Making the line even bigger (although possibly faster)

Reply:
No worries! And the `unique()` suggestion wasn't so much the line length as it might be a touch faster. You are right that the set subtraction would need to be handled differently in that case (or end up being basically the same as not doing it).

That's not important for this PR though :)